apps/frontend/src/app/3_organisms/ConversionsHistoryFrame/components/AmmConversionsHistoryFrame/AmmConversionsHistoryFrame.utils.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { Paragraph, ParagraphSize } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import {
+  BITCOIN,
+  BTC_RENDER_PRECISION,
+} from '../../../../../constants/currencies';
+import { translations } from '../../../../../locales/i18n';
+import { Swap } from '../../../../../utils/graphql/rsk/generated';
+import { dateFormat } from '../../../../../utils/helpers';
+
+export const generateRowTitle = (item: Swap) => (
+  <Paragraph size={ParagraphSize.small} className="text-left">
+    {`${t(translations.conversionsHistory.type)} - ${dateFormat(
+      item.transaction.timestamp,
+    )}`}
+  </Paragraph>
+);
+
+export const renderConversionFee = (swap: Swap) => (
+  <AmountRenderer
+    value={swap.conversionFee || '0'}
```

Comment:
Spoke with Light, you will need to add `conversionFee` and `protocolFee` as we wan't to show total fee as conversion fee